### DISCUSSION TITLE
Anyone using 1.6.0.Beta1 for your application in production?

### DISCUSSION THREAD

**Discussion by alantanlc** [mapstruct/mapstruct#3598] [Q&A] [answered]
Wondering if 1.6.0.Beta1 is safe for use in production currently as it contains a fix for Issue 3361.

**Comment by thunderhook** [comment, marked as answer]
Yes, we are already using it in production and have had no problems so far. But I guess it depends on your application. We're using a lot of mappers with many flat mappings (no deep nested mappings like `source = "a.b.c.d"`) and no subclass mappings.
